src/domain/Customer.ts

```diff
@@ -0,0 +1,8 @@
+import { Model } from 'objectmodel';
+
+export class Customer extends Model({
```

Comment:
Add address, postalCode and residence to this model

Reply:
fixed